pagina2.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Pagina 2</title>
+    <link rel="stylesheet" href= "style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- menu de nav*******************************-->
+    <header>
+        <img src="recursos/logotrain.png" alt="logo nav">
+        <div>
+            <ul>
+                <li><a href="#"></a><img src="recursos/icono-telefono.png" alt="iconotel"> 740.993.7266</li>
+                <li><a href="#"></a><img src="recursos/icono-email.png" alt="email"> Shanny90@gmail.com</li>
+                <li><a href="#"></a><img src="recursos/icono-facebook.png"></li>
+                <li><a href="#"></a><img src="recursos/icono-youtube.png"></li>
+                <li><a href="#"></a><img src="recursos/icono-google.png"></li>
+                <li><a href="#"></a><img src="recursos/icono-instagram.png"></li>
+            </ul>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="#"></a>Home</li>
+                <li><a href="#"></a>Shop</li>  
+                <li><a href="#"></a>About</li>
+                <li><a href="#"></a>Contact</li>
+                <li><a href="#"></a>Faq</li>

```

Comment:
Los anchor (`<a>`) deben contener el texto (`Home`).
```suggestion
                <li><a href="#">Home</a></li>
                <li><a href="#">Shop</a></li>  
                <li><a href="#">About</a></li>
                <li><a href="#">Contact</a></li>
                <li><a href="#">Faq</a></li>

```